index.html

```diff
@@ -43,286 +43,237 @@
+            </div>
+            <div class="history__container">
+                <div class="history__container--flex">
+                    <div class=" history__handwritten handwrittenTitle">Est. 1990</div>
+                    <img class="history__image history__img2" src="img/ourhistory.jpg"/>
                 </div>
-
-                <div class="gallery__tile-row">
-                    <li><img class="gallery__tile-item-hide" src="img/tile7.jpg"/></li>
-                    <li><img class="gallery__tile-item-hide" src="img/tile8.jpg"/></li>
+                <div class="history__container-background">
+                    <h3 class="cardHeading">Our History</h3>
+                    <p class="history__text">Founded in 1990, we have always strived to work with integrity and honesty. We are proud of our principles, which we live by every day. Wedding gift trends may change, but our values won’t.</p>
+                    <div class="history__btn button">Read Our History</div>
                 </div>
-
-            </ul>
-
+            </div>
         </section>
 
-        <section class="create-list">
+        <section class="socials">
+            <h2 class="socials__header heading">Follow Us @TheWeddingShopUK</h2>
+            <div class="socials__info">Go behind the scenes of The Wedding Shop team, discover new interior trends and be the first to know about new gifts added.</div>
+            <a class="socials__btn button" href="https://www.instagram.com/theweddingshopuk/">Follow Us</a>
+        </section>
 
-            <img class="logo__bow bow-large" src="img/Bow.png">
-            <h2 class="create-list__header heading">Start Your Wedding Gift List</h2>
-            <div class="create-list-btn button">Create a List</div>
-            <div class="create-list__chat">or, if you have any questions, <span class="underline">have a chat with one of our Gift Gurus.</span></div>
+        <section class="gallery">
+            <div class="gallery__tile">
+                <ul class="gallery__row">
+                    <li><img class="gallery__image" src="img/tile1.jpg"/></li>
+                    <li><img class="gallery__image" src="img/tile2.jpg"/></li>
+                </ul>
+                <ul class="gallery__row">
+                    <li><img class="gallery__image" src="img/tile3.jpg"/></li>
+                    <li><img class="gallery__image" src="img/tile4.jpg"/></li>
+                </ul>
+            </div>
+            <div class="gallery__tile">
+                <ul class="gallery__row">
+                    <li><img class="gallery__image" src="img/tile5.jpg"/></li>
+                    <li><img class="gallery__image" src="img/tile6.jpg"/></li>
+                </ul>
+                <ul class="gallery__row">
+                    <li><img class="gallery__image--hide" src="img/tile7.jpg"/></li>
+                    <li><img class="gallery__image--hide" src="img/tile8.jpg"/></li>
+                </ul>

```

Comment:
I'm not a fan of splitting it into multiple lists - it's one list after all, right? If you have problems with styling it you can either use the method I sent you (with flex wrap) or using :nth-child pseudo-class.